jobs/infra/playbook-jenkins.yml

```diff
@@ -342,7 +342,8 @@
     - name: update jenkins user
       user:
         name: jenkins
-        groups: jenkins,docker,lxd
+        groups: docker,lxd
+        append: yes
```

Comment:
you may notice we drop `jenkins` from groups, but that's our primary group, so we don't need to add it in the `groups` list; we just need to make sure we append the others.